integration_tests/integration_tests/test_table_monitoring_query.py

```diff
@@ -0,0 +1,138 @@
+from datetime import datetime
+from dbt_osmosis.core.osmosis import DbtProject
+from parametrization import Parametrization
+
+
+from .utils import create_test_table, insert_rows, update_var, lowercase_column_names
+
+
+@Parametrization.autodetect_parameters()
+@Parametrization.default_parameters(
+    time_bucket={"period": "day", "count": 1},
+    timestamp_column="updated_at",
+    metric_args={}
+)
+@Parametrization.case(
+    name="row_count",
+    metric="row_count",
+    input_rows=[
+        {"name": "Jessica Jones", "updated_at": "2022-01-01 13:34:29"},
+        {"name": "Luke Cage", "updated_at": "2022-01-02 13:35:28"},
+        {"name": "Luke Cage", "updated_at": "2022-01-02 15:23:28"},
+        {"name": "Luke Cage", "updated_at": "2022-01-02 17:48:28"}
+    ],
+    expected_metrics={
+        datetime(2022, 1, 2, 0, 0, 0): 1,
+        datetime(2022, 1, 3, 0, 0, 0): 3,
+        datetime(2022, 1, 4, 0, 0, 0): 0
+    }
+)
+@Parametrization.case(
+    name="row_count_no_timestamp",
+    metric="row_count",
+    timestamp_column=None,
+    input_rows=[
+        {"name": "Jessica Jones", "updated_at": "2022-01-01 13:34:29"},
+        {"name": "Luke Cage", "updated_at": "2022-01-02 13:35:28"},
+        {"name": "Luke Cage", "updated_at": "2022-01-02 15:23:28"},
+        {"name": "Luke Cage", "updated_at": "2022-01-02 17:48:28"}
+    ],
+    expected_metrics={
+        datetime(2022, 1, 4, 0, 13, 42): 4
```

Comment:
Could you define a `RUN_STARTED_AT` const at the top and use it here, and also pass it to `custom_run_started_at`? I was a bit confused a bit as to how does this work.

Reply:
Will do, you're right 👍